f5_sphinx_theme/static/js/printThis.js

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
What is this file? Why is it new? (I thought you'd already checked in PDF button stuff?)

Reply:
This module is used for PDF generation: 
https://github.com/jasonday/printThis

Yesterday, while working on my two stories, I noticed that PDF generation logic is no longer included, I suspect that it was removed/overwritten during merge.  As a result, I added it back to sphinx theme and therefore, printThis is shown as a new file

let me know if you have any questions/concerns